pkgs/applications/audio/musescore/default.nix

```diff
@@ -29,9 +29,8 @@ mkDerivation rec {
   qtWrapperArgs = [
     # MuseScore JACK backend loads libjack at runtime.
     "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libjack2 ]}"
-    # Work around crash on update from 3.4.2 to 3.5.0
-    # https://bugreports.qt.io/browse/QTBUG-85967
-    "--set QML_DISABLE_DISK_CACHE 1"
+    # There are some issues with using the wayland backend
+    "--set QT_QPA_PLATFORM xcb"
```

Comment:
So it's not possible to use it with Wayland?

Reply:
> So it's not possible to use it with Wayland?

It seems so.. Try to launch `musescore` that's from `master` with `QT_QPA_PLATFORM=wayland` and open a music sheet. Notice how the GUI is mostly unresponsive.

Reply:
Ok. Maybe this should be documented somehow? In the description maybe? Or else open an issue that tracks this problem, and link it here? Otherwise everything is looking great for me.

Reply:
Sure, found an upstream report of the issue:

https://musescore.org/en/node/321936

Reply:
Awesome!